test/threads_exec.jl

```diff
@@ -722,15 +722,68 @@ let a = zeros(nthreads())
 end
 
 # static schedule
-function _atthreads_static_schedule()
+function _atthreads_static_schedule(n)
+    ids = zeros(Int, n)
+    Threads.@threads :static for i = 1:n
+        ids[i] = Threads.threadid()
+    end
+    return ids
+end
+@test _atthreads_static_schedule(nthreads()) == 1:nthreads()
+@test _atthreads_static_schedule(1) == [1;]
+@test_throws(
+    "`@threads :static` cannot be used concurrently or nested",
+    @threads for i = 1:1; _atthreads_static_schedule(nthreads()); end,
```

Comment:
Hmm... Interesting that this causes a syntax error. It looks like an easy fix is

```suggestion
    @threads(for i = 1:1; _atthreads_static_schedule(nthreads()); end),
```

Or is there a better way?